.github/workflows/databend-release.yml

```diff
@@ -94,7 +94,7 @@ jobs:
       - name: Set musl rustflags
         if: matrix.platform == 'musl'
         run: |
-          flags="-C link-arg=-Wl,--compress-debug-sections=zlib-gabi"
+          flags="-C link-arg=-W -C target-feature=-crt-static"
```

Comment:
I need a little time to verify this

Reply:
It compiles successfully on my laptop with build-tool musl images, @everpcpc  can you help to confirm?

Reply:
Can this binary be executed successfully? I'm failed to build on my PC for:

```shell
  = note: /usr/bin/ld: /home/xuanwo/Code/datafuselabs/databend/target/x86_64-unknown-linux-musl/debug/deps/libminstant-5b533bdc94413bb4.rlib(minstant-5b533bdc94413bb4.minstant.713ea42f-cgu.13.rcgu.o): in function `std::f64::<impl f64>::ceil':
          /rustc/cd282d7f75da9080fda0f1740a729516e7fbec68/library/std/src/f64.rs:68: undefined reference to `ceil'
          /usr/bin/ld: /home/xuanwo/Code/datafuselabs/databend/target/x86_64-unknown-linux-musl/debug/deps/libminstant-5b533bdc94413bb4.rlib(minstant-5b533bdc94413bb4.minstant.713ea42f-cgu.13.rcgu.o): in function `std::f64::<impl f64>::round':
          /rustc/cd282d7f75da9080fda0f1740a729516e7fbec68/library/std/src/f64.rs:88: undefined reference to `round'
          collect2: error: ld returned 1 exit status
```

But I think it's not related to this PR, instead, it's resulted from https://github.com/rust-lang/rust/issues/61328

It's better to run our test suits on musl binaries too (maybe in a new PR).

Reply:
@Xuanwo try with build-tool image?
`IMAGE='datafuselabs/build-tool:x86_64-unknown-linux-musl' RUSTFLAGS='-C link-arg=-Wl,--compress-debug-sections=zlib-gabi -C target-feature=-crt-static' ./scripts/setup/run_build_tool.sh cargo build --target x86_6
4-unknown-linux-musl`